### PR TITLE
ref(tsc): convert static/app/views/discover\/tableView.spec to tsx

### PR DESCRIPTION
**Pull request by JonasBa**
Convert tableView.spec to tsx

### CODE REVIEW

File: static/app/views/discover/table/tableView.spec.tsx [hunk 1]
```diff
@@ -404,26 +409,30 @@ describe('TableView > CellActions', function () {
...
-              title: null,
               'p99(measurements.custom.kilobyte)': 'kilobyte',
             },
           },
         }}
         onChangeShowTags={onChangeShowTags}
       />
     );
-    await userEvent.hover(screen.getByText('444.3 KB'));
+    await userEvent.hover(screen.getByText(/444\.3 KB/));
```
Comment (priscilawebdev) [inline static/app/views/discover/table/tableView.spec.tsx:432]: why this change?

Comment (JonasBa) [reply static/app/views/discover/table/tableView.spec.tsx:432]: I'll revert it back, at some point the test broke and I was trying to see if the value was printed somewhere else
